util.sh

```diff
@@ -388,6 +388,30 @@ function aws_ecs_redeploy {
     )
 }
 
+# redeploy existing lambda
+# usage:
+# aws_ecs_redeploy <image_uri>
+function aws_lambda_redeploy_by_image {
+    image_uri=$1
```

Comment:
could we outline a scenario where this will be getting run and the exact steps for it? Its unclear to me if we would be invoking this from within the aws/ repo or from any other repo were we have lambdas.

Reply:
will ping in slack as this repo is public